internal/pkg/service/stream/api/service/sink.go

```diff
@@ -395,6 +395,19 @@ func (s *service) ListSinkVersions(ctx context.Context, scope dependencies.SinkR
 	return s.mapper.NewSinkVersions(ctx, formatAfterID(payload.AfterID), payload.Limit, list)
 }
 
+func (s *service) SinkVersionDetail(ctx context.Context, scope dependencies.SinkRequestScope, payload *api.SinkVersionDetailPayload) (res *api.Version, err error) {
+	if err := s.sinkMustExist(ctx, scope.SinkKey()); err != nil {
+		return nil, err
+	}
+
+	sink, err := s.definition.Sink().Version(scope.SinkKey(), payload.VersionNumber).Do(ctx).ResultOrErr()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
```suggestion
	sinkKey := scope.SinkKey()
	if err := s.sinkMustExist(ctx, sinkKey); err != nil {
		return nil, err
	}

	sink, err := s.definition.Sink().Version(sinkKey, payload.VersionNumber).Do(ctx).ResultOrErr()
	if err != nil {
		return nil, err
	}
```
This simplifies the access to key. It can be called frequently. Maybe adjust also code in previous PRs with source and sink?

Reply:
we are using it in all cases, I see no reason why I should change it. :)

Reply:
agreed, I don't think this is necessary